### PR TITLE
Updating server dependancies.

### PR DESCRIPTION
**Pull request by crspeller**

### CODE REVIEW

File: glide.yaml [hunk 1]
```diff
@@ -81,13 +82,16 @@ import:
...
 - package: gopkg.in/olivere/elastic.v5
-  version: v5.0.43
+  version: v5.0.45
 - package: github.com/mattermost/gorp
   version: 995ddf2264c4ad45fbaf342f7500e4787ebae84a
 - package: github.com/go-redis/redis
-  version: v6.5.2
+  version: v6.5.7
 - package: github.com/stretchr/testify
-  version: ~1.1.4
```
Comment (ccbrown) [inline glide.yaml]: So what's the process for updating these? Are you going through these manually? I'm used to specified a range based on compatibility promises and just using `glide up`. Not really sure how it works when you specify exact versions.

**Comment by coreyhulen** [comment]
@crspeller I think you need a rebase to fix the broken master.

**Comment by crspeller** [comment]
Rebased.

File: glide.yaml (hunk 1 above)
Comment (crspeller) [inline glide.yaml]: No, just using `glide cw` to find the latest versions.

**Pull request merged**